### ISSUE TITLE
[General] Can't share a completed quest

### ISSUE THREAD

**Issue by nimeral** [SunwellTracker/issues#530]
Decription: can't share a completed quest. 
In general (it's another issue), I feel that way too many simple quests can't be shared at all, but I don't know how to fix it without reporting all them one by one - and there're dozens (e.g. Tinkmaster Overspank, Brother Anton)

How it works: if you've completed the quest, you can't share it

How it should work: you should be able to share a shareable quest no matter what

Source (you should point out proofs of your report, please give us some source): https://imgur.com/a/WLGDx you can see that I press "Share" and nothing happens; the quest itself is shareable as I checked before

**Comment by MrTide** [comment]
hi you need to provide evidence that it should be possible

**Comment by artemmavrich** [comment]
It must not be possible as i know, and on retail wow i was playing and that was impossible

**Comment by nimeral** [comment]
Would this http://wow.allakhazam.com/wiki/share_quest_%28WoW%29 count as a proof? Note that it's not "The questgiver has supplied you with an item that is needed to complete the quest" situation (such quests are of course not shareable).

**Comment by MrTide** [comment]
Quest sharing can fail if any of the following situations occurs:

The other player is too far away. The range on quest sharing is very short.
The other player's quest log is full.
The other player has completed the quest already.
The other player is inelgibile for the quest, either because they are too low in level or because they must finish prerequisite quests first.

"quest sharing fails if the other player has completed the quest already "

**Comment by nimeral** [comment]
>quest sharing fails if the other player has completed the quest already

Well, it's not this situation. But I'll try to screenshot something similar so it's more clear.

I've realized that it's likely to be the root of all sharing issues I've encountered. And I've encountered a few for I'm leveling in a pair. 

Searching for the Lost Satchel (5722), The Missing Diplomat (1274), In Search of Thaelrid (1198), Tran'rek (2864), Tinkmaster Overspark (2923), Brother Anton (6141). All these quests are simply "go there and talk to a guy/corpse", and I think I was unable to share any of them. Maybe not because or wrong flags, maybe simply because they're Completed from the beginning?
